pom.xml

```diff
@@ -445,6 +446,25 @@
         </dependency>
 
         <!--junit for unit test-->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit-jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>${junit-jupiter.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <!--JUnit Jupiter Engine to depend on the JUnit4 engine and JUnit 4 API  -->
+        <dependency>
+            <groupId>org.junit.vintage</groupId>
+            <artifactId>junit-vintage-engine</artifactId>
+            <version>${junit-jupiter.version}</version>
```

Comment:
scope should be test?

Reply:
snyk merge gates are failing. please make sure they pass before pushing.

Reply:
I think the snyk issue is not fixed, we may need to disable it for now

Reply:
Good catch! Snyk is passing after this change so probably it ignores test dependencies.